functions/help.php

```diff
@@ -0,0 +1,66 @@
+<?php
+	include("init.php");
+
+	if($_SERVER['REQUEST_METHOD']=="POST"){
+
+		// Inputs
+		$name = trim($_POST['name']);
+		$useremail = trim($_POST['useremail']);
+		$sub = test_input($_POST['subject']);
+		$body = test_input($_POST['body']);
+
+		// Verification
+		$error = array();
+		if(empty($name)){
+			$error[] = "Require name";
+		}
+		else if (!preg_match("/^[a-zA-Z ]*$/",$name)) {
```

Comment:
this may create a problem as we are not checking at the time of sign up if the user name has only letters or white space, remove this check